tests/scenarios/templates/execution_report_data.sparql

```diff
@@ -0,0 +1,54 @@
+{#
+  This file contains the template for the execution report results
+  :param: context_name_graph
+#}
+
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX shacl: <http://www.w3.org/ns/shacl#>
+
+SELECT DISTINCT ?s ?p ?o
+{% if context_name_graph %}
+FROM <{{context_name_graph}}>
+{% endif %}
+WHERE {
+  {
+    # Find the subject with rdf:type shacl:ValidationReport
```

Comment:
there must be a smarter way using property paths to achieve the same

Reply:
There was a solution that used a blank node to traverse through all layers untill no other property was found for a subject but graphDB could not handle this query 

```sparql
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX shacl: <http://www.w3.org/ns/shacl#>

SELECT DISTINCT ?connectedNode
FROM <urn:traversal-harvesting:dereference_test1_sparql.yml>
WHERE {
  # Find the subject with rdf:type shacl:ValidationReport
  ?subject rdf:type shacl:ValidationReport .
  
  # Get all nodes connected to this subject through any path
  ?subject (<>*) ?connectedNode .
}
```

more specifically the "<>*" part.
 the SPARQL endpoint you are using does not support the shorthand <> syntax for any property.
 
 Therefore I had to resolve the issue by manually figuring out the total deph by these union blocks

Reply:
think the typical syntax is `<source> (<>|!<>)* <target>` 
or `<source> (:|!:)* <target>` if you can add a default prefix declaration `@prefix : <someurl>`

have not checked the above with graphdb